src/main.rs

```diff
@@ -439,7 +452,12 @@ fn main() {
         (None, None)
     };
 
-    let output = matches.value_of("output").unwrap();
+    let output = matches.value_of("output").unwrap().to_string();
+    let output_is_dir = PathBuf::from(output.clone()).is_dir();
+    if !output.is_empty() && !output_is_dir {
```

Comment:
We can adapt according to the kind of path we've and the tool we're using: I mean `--metrics -o not_dir` is forbidden but can be allowed with `--foo -o not_dir`.
So just disallow file type when we're running metrics.

Reply:
You're right, I will change that, thanks!